lightning/src/routing/network_graph.rs

```diff
@@ -1008,11 +1008,11 @@ impl NetworkGraph {
 		Ok(())
 	}
 
-	/// Close a channel if a corresponding HTLC fail was sent.
+	/// Marks a channel in the graph as failed if a corresponding HTLC fail was sent.
 	/// If permanent, removes a channel from the local storage.
 	/// May cause the removal of nodes too, if this was their last channel.
 	/// If not permanent, makes channels unavailable for routing.
-	pub fn close_channel_from_update(&self, short_channel_id: u64, is_permanent: bool) {
+	pub fn fail_channel(&self, short_channel_id: u64, is_permanent: bool) {
```

Comment:
suggestion: prefer `channel_failed` because imo the `gossip` isn't the one failing the channel

Reply:
Also renamed `fail_node`.